defaults/main.yml

```diff
@@ -295,7 +301,7 @@ atom_npm_version: >-
   {{
     (
       "10.9.2" if ansible_os_family == 'Debian' and ansible_distribution_version >= '22.04' else
-      "10.9.2" if ansible_os_family == 'RedHat'  and ansible_distribution_version > '7' else
+      "10.9.2" if ansible_os_family == 'RedHat' and atom_php_version | int >= 80 else
```

Comment:
Confirming for RHEL 9 deployment with atom_php_version = "74", above results in
`TASK [artefactual.atom : Update npm to 9.8.1 version]`
as expected. Approved!